src/main/java/com/google/api/codegen/viewmodel/StaticLangApiView.java

```diff
@@ -61,6 +61,9 @@
 
   public abstract boolean hasLongRunningOperations();
 
+  @Nullable // Used in Java
+  public abstract String stubInterfaceName();
```

Comment:
Nullable feels a little icky. I generally think "NOT IMPLEMENTED" string is better. Looks like you'll have to change all LanguageTransformers though... Up to you.